demo/js/welcome.js

```diff
@@ -9,6 +9,8 @@
   }
 
   function setupShepherd() {
+    var prefix = 'demo-';
+    var buttonSecondaryClass = prefix + 'shepherd-button-secondary';
```

Comment:
This is the demo code, we shouldn't need to do anything here.

Reply:
we have styling for this class as part of the library here: https://github.com/shipshapecode/shepherd/blob/master/src/js/styles/generateStyles.js#L46

So it's not a regular custom class like other 'class 1' class 2', etc
So it should be prefixed as well.